knowledge_repo/postprocessors/format_checks.py

```diff
@@ -12,7 +12,9 @@
 
 OPTIONAL_FIELD_TYPES = {
     'path': str,
-    'updated_at': datetime.datetime
+    'updated_at': datetime.datetime,
+    'start_publishing': bool,   # If false, this post starts out private
```

Comment:
I thought we were going for `private` rather than `start_publishing`. Did something change?